modules/my_health/app/controllers/my_health/v1/all_triage_teams_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module MyHealth
+  module V1
+    class AllTriageTeamsController < SMController
+      def index
+        resource = client.get_all_triage_teams(@current_user.uuid, use_cache?)
+        raise Common::Exceptions::InternalServerError if resource.blank?
```

Comment:
We should eventually fix this line, which we have in several places in both SM and MR, per https://dsva.slack.com/archives/C018V2JCWRJ/p1694120211778619

Reply:
oh, that is right. I recall now. Yes, I think we can do an overall sweep in a separate PR. I would like to get your advice on what exactly needs to be done here 👍🏻